### PR TITLE
feat: save alarm configs in firebase

### PR DESCRIPTION
**Pull request by rumeshwick**

### CODE REVIEW

File: src/components/Alarm/Alarm.tsx [hunk 1]
```diff
@@ -87,13 +90,8 @@ const Alarm: FC<Props> = (props: Props) => {
   };
 
   useEffect(() => {
-    /**
-     * Alarms polling dispatch action will be fired when the component is mounted
-     */
-    props.startUpdateAlarms();
     return onUnmount;
```
Comment (man-csl) [inline src/components/Alarm/Alarm.tsx:93]: Is this a good practice?

File: src/components/Alarm/Configurator/AlarmConfigurator.tsx [hunk 2]
```diff
@@ -104,13 +94,20 @@ const AlarmConfigurator: FC<Props> = (props: Props) => {
...
+    props.setAlerts({
+      type: 'error',
+      text: msg,
+      hideApp: false,
+    });
+  };
+
   /**
    * This will open the configurator popup on alarm settings icon click.
    * Alarms polling will be stopped when the popup opens.
```
Comment (man-csl) [inline src/components/Alarm/Configurator/AlarmConfigurator.tsx]: is this applicable now?

File: src/components/Alarm/Configurator/AlarmConfigurator.tsx [hunk 3]
```diff
@@ -121,42 +118,35 @@ const AlarmConfigurator: FC<Props> = (props: Props) => {
    * This will close the popup and start polling alarms
```
Comment (man-csl) [inline src/components/Alarm/Configurator/AlarmConfigurator.tsx]: is this correct?

File: src/components/Alarm/Alarm.tsx (hunk 1 above)
Comment (rumeshwick) [reply src/components/Alarm/Alarm.tsx:93]: In react this is how we can execute a function on component unmount. We have to implement that as a return function in useEffect hook

File: src/components/Alarm/Configurator/AlarmConfigurator.tsx (hunk 2 above)
Comment (rumeshwick) [reply src/components/Alarm/Configurator/AlarmConfigurator.tsx]: Fixed the comment issue

File: src/components/Alarm/Configurator/AlarmConfigurator.tsx (hunk 3 above)
Comment (rumeshwick) [reply src/components/Alarm/Configurator/AlarmConfigurator.tsx]: Fixed the comment issue

**Review by man-csl — APPROVED** [review]

**Pull request merged**